pcstac/tests/resources/test_item.py

```diff
@@ -490,7 +490,7 @@ async def test_field_extension_exclude_default_includes(app_client):
 async def test_search_intersects_and_bbox(app_client):
     """Test POST search intersects and bbox are mutually exclusive (core)"""
     bbox = [-118, 34, -117, 35]
-    geoj = Polygon.from_bounds(*bbox).__geo_interface__
+    geoj = Polygon.from_bounds(*bbox).dict(exclude_none=True)
```

Comment:
shapely was a big test dependency, while geojson_pydantic is already available. (this change was made so I can run CI on arm64 platform)